R/make-objects.R

```diff
@@ -243,11 +243,16 @@ make_exemplar_model_from_data = function(
     group = NULL,
     category = "category",
     cues,
-    sim_function = function(x, y) {
-      j <- 2
-      k <- 1
-      distance <- (x - y)^j
+    sim_function = function(x, y, j = 2, k = 1) {
+      # If j = 1, this uses Manhattan city block distances
+      # If j = 2, this uses Euclidean distances
+      distance <- sum((x - y)^j)^(1/j)
```

Comment:
I just realised that Euclidean distance is just the Pythagorean distance. If I had known this I wouldn't have had to be so mystified all this time.